packages/webview_flutter/webview_flutter_android/lib/webview_android_widget.dart

```diff
@@ -174,6 +174,43 @@ class WebViewAndroidPlatformController extends WebViewPlatformController {
     return webView.loadUrl(url, headers ?? <String, String>{});
   }
 
+  @override
+
+  /// Makes a specific HTTP request ands loads the response in the webview.
+  ///
+  /// [WebViewRequest.method] must be one of the supported HTTP methods
+  /// in [WebViewRequestMethod].
+  ///
+  /// If [WebViewRequest.headers] is not empty, its key-value pairs will be
+  /// added as the headers for the request.
+  ///
+  /// If [WebViewRequest.body] is not null, it will be added as the body
+  /// for the request.
+  ///
+  /// Throws an ArgumentError if [WebViewRequest.uri] has empty scheme.
+  ///
+  /// Android only:
```

Comment:
This paragraph is the only part that should stay. This is the Android package though, so you don't need "Android only".